google/cloud/bigquery/_helpers.py

```diff
@@ -26,7 +26,7 @@
 from google.cloud._helpers import _RFC3339_MICROS
 from google.cloud._helpers import _RFC3339_NO_FRACTION
 from google.cloud._helpers import _to_bytes
-import pkg_resources
+import packaging.version
```

Comment:
I wanted to add type annotations, but it turns out `pkg_resources` is just a thin wrapper around `packaging`.

https://github.com/pypa/setuptools/blob/a4dbe3457d89cf67ee3aa571fdb149e6eb544e88/pkg_resources/__init__.py/#L112